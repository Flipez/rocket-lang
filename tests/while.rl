a = 0
while (a != 3) 
  puts(a)
  a = a + 1
end
