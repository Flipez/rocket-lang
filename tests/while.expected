0
1
2
null
